Maps.resources.php

```diff
@@ -16,6 +16,7 @@
 	$pathParts = ( explode( DIRECTORY_SEPARATOR . 'extensions' . DIRECTORY_SEPARATOR, __DIR__, 2 ) );
 
 	$moduleTemplate = array(
+		'position' => 'top',
```

Comment:
This is a more complex setup than usual. However I guess this is the right spot since it applies to all modules using the template.